src/components/ReactSlider/ReactSlider.md

```diff
@@ -57,6 +88,26 @@ Vertical slider
 />
 ```
 
+Vertical slider with marks
+
+```jsx
+<ReactSlider
+    className="vertical-slider"
+    thumbClassName="example-thumb"
+    trackClassName="example-track"
+    defaultValue={[0, 5, 10]}
+    max={10}
+    ariaLabel={['Lowest thumb', 'Middle thumb', 'Top thumb']}
+    renderThumb={(props, state) => <div {...props}>{state.valueNow}</div>}
+    orientation="vertical"
+    invert
+    pearling
+    minDistance={1}
+    marks={[0, 3, 5, 8, 10]}
```

Comment:
The spacing of these marks in this example is a little strange, maybe just `[0, 5, 10]`?

Reply:
the idea is to show some marks also away from the thumbs otherwise they just cover them, but as you like